loom/common/core/performance.cpp

```diff
@@ -530,16 +547,21 @@ static void LoomProfilerEntryDumpRecurse(LoomProfilerEntry *data, char *buffer,
     else
     {
         // dump out this one:
-        lmLogError(gProfilerLogGroup, "%7.3f %7.3f %8d %s%s",
+        lmLogError(gProfilerLogGroup, "%8.3f %8.3f %8.3f %8.3f %8.3f %8d %s%s",
                    data->mRoot == NULL ? 100.0 : 100.0 * data->mTotalTime / totalTime,
                    data->mRoot == NULL ? 100.0 : 100.0 * (data->mTotalTime - data->mSubTime) / totalTime,
+                   data->mTotalTime / (1000.0 * 1000.0 * (data->mInvokeCount > 0 ? data->mInvokeCount : 1)),
+                   data->mMaxTime / (1000.0 * 1000.0),
```

Comment:
These numbers are screwy on non-Android platform (ie they are proportionally right but don't show ms). Can we include a disclaimer in the profiler output for non-Android platforms? ie just a "Total/max may not be expressed in milliseconds on non-Android devices."?